gherkin-lint/javascript/TODO.md

```diff
@@ -0,0 +1,6 @@
+# TODO
+
+* [ ] Command line interface in `bin/gherkin-lint`
+  * [ ] Write events to `STDOUT` (consumed by other consumers/formatters)
+* [ ] Change `GherkinLinter` constructor to take a list of Rule objects
+* [ ] Create a Rule factory that build rules from a config file
```

Comment:
Should we add 
`[ ] Allow rules to be configured for different levels (eg error ,warning)`

This is something that I got a request for in my gherkin linter and it's in par with other linters, like eslint.